### PR TITLE
Fix formatting on Windows

### PR DESCRIPTION
**Pull request by princemaple**
Alternative PR to #373

### CODE REVIEW

**Comment by princemaple** [comment]
🤕 String.replace that takes a replacement function was added in Elixir 1.9

**Comment by lukaszsamson** [comment]
> 🤕 String.replace that takes a replacement function was added in Elixir 1.9

We've just dropped support for elixir 1.7 in 0.6. I wouldn't be happy with dropping 1.8 so soon.

**Comment by princemaple** [comment]
When Elixir 1.11 is out, you can probably drop 1.8. But now, yeah, I agree with you - hence the 🤕 .

**Comment by princemaple** [comment]
Gonna try a new idea.

File: apps/language_server/lib/language_server/server.ex [hunk 1]
```diff
@@ -815,7 +815,7 @@ defmodule ElixirLS.LanguageServer.Server do
 
   defp set_project_dir(%{project_dir: prev_project_dir, root_uri: root_uri} = state, project_dir)
        when is_binary(root_uri) do
-    root_dir = SourceFile.path_from_uri(root_uri)
+    root_dir = root_uri |> SourceFile.path_from_uri() |> Path.absname()
```
Comment (princemaple) [inline apps/language_server/lib/language_server/server.ex:818]: this normalizes the `project_dir` path, so it's in "elixir case" (lower `c:/`)

**Comment by princemaple** [comment]
@lukaszsamson @axelson this patch should be mergeable now!

- passes test (works on Elixir 1.8)
- works on my machine - Windows
- does not break case sensitive systems

**Review by axelson — APPROVED** [review]
Thanks! This looks good now. :heart:

**Pull request merged**